.github/workflows/release.yml

```diff
@@ -22,19 +22,15 @@ jobs:
     steps:
       - uses: actions/checkout@v4
 
-      - uses: actions/setup-python@v5
+      - name: Install uv
+        uses: astral-sh/setup-uv@v3
         with:
-          python-version: "3.13"
-          allow-prereleases: true
-
-      - name: Install dependencies
-        run: |
-          python -m pip install -U pip uv
-          python -m uv pip install --system hatch
+          enable-cache: true
+          version: "0.4.x"
 
       - name: Build package
         run: |
-          hatch build
+          uv build
```

Comment:
👀  I haven't used this yet, but I'm perplexed by it now.

Reply:
🤷‍♂️ giving it a try, I haven't either yet

Reply:
I should say, I haven't used it when cutting an actual release, but it works fine locally so I'm rolling with it.